src/includes/gateways/CreditPayment.php

```diff
@@ -261,21 +261,26 @@ protected function get_installments()
     if($this->is_single_order())
       return $this->installments;
 
+    $installments = 0;
+
     foreach($this->vindi_settings->woocommerce->cart->cart_contents as $item) {
-      $plan_id = $item['data']->get_meta('vindi_subscription_plan');
-      if (!empty($plan_id))
-        break;
+      $plan_id = $item['data']->get_meta('vindi_plan_id');
+      
+      if (!empty($plan_id)) {
+        $plan = $this->routes->getPlan($plan_id);
+        
+        if($installments == 0) {
+          $installments = $plan['installments'];
+        }
+        elseif($plan['installments'] < $installments) {
+          $installments = $plan['installments'];
+        }
```

Comment:
Apenas por questões estéticas o que acha da seguinte alteração? 💅🏾 

```suggestion
        if ($installments == 0) {
          $installments = $plan['installments'];
        } elseif ($plan['installments'] < $installments) {
          $installments = $plan['installments'];
        }
```